README.md

```diff
@@ -87,11 +87,20 @@ Accessibility should be incorporated and tested throughout the design and develo
 
 ## Policy & Informal
 
-* **Analytics:** All of our sites should have code snippets for 18F and Digital Analytics Program Google Analytics as well as Google Webmaster Tools.
 * Agency or Organization Communications Office
 * Legislative Affairs
 * Office of General Counsel
 
+### Analytics
+
+All of our sites should have code snippets for
+
+* 18F Google Analytics (18F only)
```

Comment:
@konklone @afeld ... our general policy (while not perfect) has been 18F analytics for stuff that 18F supports and DAP on everything. Does that make sense? 

(On another note, is there a reason analytics.usa.gov only has DAP? I would think it would have 18F as well. This is probably not the right place to discuss this. ;-) )